docs/changelog.md

```diff
@@ -35,6 +35,7 @@ permalink: /changelog/
     })
   }
   ```
+  * ⚠️ **This is a breaking change**: Redirect source and target flags are now passed to history tracking delegates. As part of this change, `HistoryTrackingDelegate.onVisited()` receives a new `PageVisit` data class as its second argument, specifying the `visitType` and `redirectSource`. For more details, please see [PR #4268](https://github.com/mozilla-mobile/android-components/pull/4268).
```

Comment:
@linacambridge looks like you're missing the first line and accidentally add the '+'.  This is causing the ktlint issue.

Reply:
Hmm, not sure what you mean. Could you push a follow-up commit or a GitHub suggested change with the style you'd like, please?

Reply:
My mistake, I thought this was missing the title.  After expanding, I see that you're putting this behind a existing **engine** change.  Once the tests pass I'll approve.  Thanks!